daemon/src/wallet.rs

```diff
@@ -135,6 +135,8 @@ impl Actor<ElectrumBlockchain> {
 
 impl Actor<ElectrumBlockchain> {
     fn sync_internal(&mut self) -> Result<WalletInfo> {
+        self.wallet.ensure_addresses_cached(1000)?;
```

Comment:
perhaps it would be worthwhile to expose this as an optional CLI parameter? In case we run into trouble with calculating wallet balance, we would be able to restart with a little tweak to the parameters instead of cutting a new release

Reply:
I dont' think this is necessary. I'd rather make this dynamic and increase slowly until this maximum is reached.